setup-k3d/action.yaml

```diff
@@ -114,6 +114,24 @@ runs:
         # K3d sets this up for us in the node, but we're responsible for the host
         sudo echo "127.0.0.1 ${{ inputs.registry-host }}" | sudo tee -a /etc/hosts
 
+        # Make the CNI bin directory predictable.
```

Comment:
can we make this configurable via the action input?

Reply:
This is lower level than most workflow ever needs, and usually the same configuration is available at the CNI installer (as CNI bin directory is very dependent on the K8s distro). I can add this configuration when we need it?

Reply:
sure

Reply:
One thing that probably isn't clear here is that we are not doing any change to k3s to expose this path. The paths that the PodSandbox uses is still /var/lib/rancher/k3s/data/${uuid}/bin.

This only allows the CNI installers to know where to copy their binary in.